### DISCUSSION TITLE
iframe support

### DISCUSSION THREAD

**Discussion by ablyankitc** [mynaparrot/plugNmeet-server#313] [Q&A] [answered]
does it supports opening in a iframe?

**Comment by jibon57** [comment]
It doesn't need iframe. You can read here: https://www.plugnmeet.org/docs/api/join_room

**Reply by jibon57** [reply, marked as answer]
You can check example from here: https://github.com/mynaparrot/plugNmeet-sdk-php/blob/main/examples/conference.php

**Comment by dvkazin** [comment]
In case, anyone else needs this.

```
<iframe
	src="meet.mydomen.org"
	style="width: 100vw; height: 100dvh; border: none;"
	allow="camera; microphone; display-capture;">
</iframe>
```
